src/actor/ActorTemplate.js

```diff
@@ -205,34 +255,42 @@ exports = function(inherits) {
     }
 
     /**
-      * Set {@link Actor#vx} and {@link Actor#vy} to aim for the specified point, with the specified speed.
-      * @func Actor#headToward
-      * @arg {number} x
-      * @arg {number} y
-      * @arg {number} speed
-      */
+     * Set vx and vy to aim for the specified point, with the specified speed.
+     * @method ActorTemplate#headToward
+     * @param  {number} x
+     * @param  {number} y
+     * @param  {number} speed
+     */
     this.headToward = function(x, y, speed) {
       var targetAngle = Math.atan2(y - this.y, x - this.x);
       this.vx = speed * Math.cos(targetAngle);
       this.vy = speed * Math.sin(targetAngle);
     };
 
+    /**
+     * Rotate the actor at the specified point with the specified rotation offset
```

Comment:
`rotateAt` should probably be `rotateToward`. The preposition `at` could be misconstrued as implying a specific anchor point. This function appears in other API's, and it does indeed refer to the point around which something is rotating, i.e. the anchor: https://msdn.microsoft.com/en-us/library/system.drawing.drawing2d.matrix.rotateat(v=vs.110).aspx